assets/eip-3475/ERC3475.sol

```diff
@@ -49,10 +49,9 @@ contract ERC3475 is IERC3475 {
         _classMetadata[0].title = "symbol";
         _classMetadata[0]._type = "string";
         _classMetadata[0].description = "symbol of the class";
+       
+       
         _classes[0]._values[0].stringValue = "DBIT Fix 6M";
-        _classMetadata[1].title = "symbol";
```

Comment:
If you want to edit EIP-3475's files, you'll need to make a separate PR for that.

Reply:
```bash
git checkout origin/master -- assets/eip-3475/ERC3475.sol
```